data/plugins/projectsearch.lua

```diff
@@ -332,3 +332,5 @@ keymap.add {
   ["home"]               = "project-search:move-to-start-of-doc",
   ["end"]                = "project-search:move-to-end-of-doc"
 }
+
+return {}
```

Comment:
Should we return the `ResultsView` here?

Reply:
Ideally we'd have a table that contains resultsview, that represents the plugin as a whole, like the other plugins. I'll change it to do that.

Reply:
Actually, seems like most of the core plugins don't bother doing this; so I'll just return to that convention.